libr/bin/format/mach0/coresymbolication.c

```diff
@@ -164,7 +164,7 @@ RCoreSymCacheElement *r_coresym_cache_element_new(RBinFile *bf, RBuffer *buf, ut
 	if (!hdr) {
 		return NULL;
 	}
-	if (hdr->version != 1) {
+	if (hdr->version != 1 && hdr->version != 7) {
 		eprintf ("Unsupported CoreSymbolication cache version (%d)\n", hdr->version);
```

Comment:
use R_LOG_ERROR instead of eprintf

```suggestion
		R_LOG_ERROR ("Unsupported CoreSymbolication cache version (%d)", hdr->version);
```

Reply:
done